### PR TITLE
Clean empty work external identifiers

### PR DESCRIPTION
**Pull request by amontenegro**

### CODE REVIEW

**Comment by amontenegro** [comment]
[Remove empty external identifiers from works](https://trello.com/c/JHw1VzvM/3145-remove-empty-external-identifiers-from-works)

File: orcid-persistence/src/main/resources/db/updates/fix_empty_work_external_ids.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		<sql>update empty_work_identifiers set fixed_external_ids_json = (replace(fixed_external_ids_json::text, '{"relationship":"SELF","url":{"value":""},"workExternalIdentifierType":null,"workExternalIdentifierId":null}', ''))::json;</sql>
+		<sql>update empty_work_identifiers set fixed_external_ids_json = (regexp_replace(fixed_external_ids_json::text, '{"relationship":"SELF","url":{"value":""},"workExternalIdentifierType":".*","workExternalIdentifierId":null}', ''))::json;</sql>
+		<sql>update empty_work_identifiers set fixed_external_ids_json = (regexp_replace(fixed_external_ids_json::text, '{"relationship":"SELF","url":null,"workExternalIdentifierType":".*","workExternalIdentifierId":null}', ''))::json;</sql>
+		<sql>update work as w set external_ids_json=ewi.fixed_external_ids_json, last_modified=now() from empty_work_identifiers as ewi where w.work_id = ewi.work_id and w.orcid = ewi.orcid;</sql>
+		<sql>drop table if exists empty_work_identifiers;</sql>
```
Comment (rcpeters) [inline orcid-persistence/src/main/resources/db/updates/fix_empty_work_external_ids.xml]: @amontenegro no reindexing records?

Comment (amontenegro) [inline orcid-persistence/src/main/resources/db/updates/fix_empty_work_external_ids.xml]: Fixed

**Comment by amontenegro** [comment]
Fixed @rcpeters

**Pull request merged**